docs/source/user_guide/release_notes.md

```diff
@@ -1,64 +1,176 @@
 # Release note
 
+## v0.9.1rc2 - 2025.08.04
+This is the 2nd release candidate of v0.9.1 for vLLM Ascend. Please follow the [official doc](https://vllm-ascend.readthedocs.io/en/v0.9.1-dev/) to get started.
+
+### Highlights
+* DeepSeek w4a8 quantization support: [#1320](https://github.com/vllm-project/vllm-ascend/pull/1320) [#1910](https://github.com/vllm-project/vllm-ascend/pull/1910)
```

Comment:
Combine DeeSeek and Qwen w4a8. We can rewrite to MOE and dense w4a8 quantization support